src/Lesson2/HomeWorkLesson2.java

```diff
@@ -0,0 +1,54 @@
+package Lesson2;
+
+public class HomeWorkLesson2 {
+    public static void main(String[] args) {
+
+        boolean j = inRage(5, 100);
+        System.out.println(j);
+        nunber(-5);
+        boolean h = plusMinus(-5);
+        System.out.println(h);
+        printLine("test", 3);
+        boolean x = isLeap(2300);
+        System.out.println(x);
+    }
+
+    // Задание 1
+    public static boolean inRage(int a, int b) {
+        int sum = a + b;
+        boolean result;
+        if (sum >= 10 && sum <= 20) result = true;
+        else result = false;
+        return result;
```

Comment:
поскольку логическое выражение само по себе возвращает boolean результат, то можно упростить:
```suggestion
        return (sum >= 10 && sum <= 20);
```